pkg/reconciler/workload/apibindinglabel/apibindinglabel_indexes.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apibindinglabel
+
+import (
+	"fmt"
+
+	"github.com/kcp-dev/logicalcluster/v2"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	"github.com/kcp-dev/kcp/pkg/client"
+)
+
+const indexAPIBindingsByWorkspaceExport = ControllerName + "apiBindingsByWorkspaceExport"
+
+// indexAPIBindingsByWorkspaceExportFunc is an index function that maps an APIBinding to the key for its
+// spec.reference.workspace.
+func indexAPIBindingsByWorkspaceExportFunc(obj interface{}) ([]string, error) {
```

Comment:
Isn't it already available in global indexers here: https://github.com/qiujian16/kcp/blob/c8d456da43271170975228b4e55af03173be2639/pkg/indexers/apibinding.go#L97 ?